talisker/gunicorn.py

```diff
@@ -170,6 +182,8 @@ def init(self, parser, opts, args):
             'loglevel': 'DEBUG',
         })
 
+        cfg['worker_exit'] = prometheus_multiprocess_worker_exit
```

Comment:
I think this should probably only be added to cfg iff (workers >1 and prometheus_multiproc_dir in os.environ and util.pkg_is_installed('prometheus_client'))